### ISSUE TITLE
Coordinate surface tagging on roadway and crossing

### ISSUE THREAD

**Issue by watmildon** [tsmock/sidewalks#13]
My current work pattern is to add surface tagging and fix alignment of roads, then work on sidewalks. Occasionally I will miss a roadway surface tag and it (obviously) won't get picked up on the crossing node. Maybe it's better as a validator step but it could be nice to have surface tags from roadways proactively applied to crossings.

**Comment by tsmock** [comment]
This is a harder problem to solve.
Yes, I could do that (and it _might_ make sense for crossings without a surface tag), but I would not want to copy a conflicting surface tag over.

Realistically, I can justify copying the surface tag from a highway to the crossing on crossing creation -- the user is more likely to fix the surface data at that time.

Copying the surface tag from the highway to the crossing after the fact is a bit harder to justify. I'd rather have the user look at each crossing individually, instead of mass editing crossings.

I'd rather have _no_ information than _wrong_ information, and I think it is substantially more likely for that to happen when copying surface data from a highway to a crossing when adding the surface to a highway.

In other words, a validator rule would be more appropriate.

EDIT: Removing `surface` tagging when changing a highway surface _might_ be appropriate though.

**Comment by watmildon** [comment]
That's totally reasonable. I'll try and pay attention and see if there are any unambiguous cases but it does seem to be tricky.

**Comment by aighes** [comment]
How about display a selection form (like the one from  one-click-highway) if there is no surface existing?

<img width="338" height="270" alt="Image" src="https://github.com/user-attachments/assets/2ba73118-0f17-4601-a3b3-923a16b30a38" />
